packages/sdk-react/src/components/receivables/ReceivablesTable/ReceivablesTable.stories.tsx

```diff
@@ -16,7 +17,76 @@ export const FullPermissions: Story = {
   },
   render: (args) => (
     <div style={{ height: 500, padding: 20 }}>
-      <ReceivablesTable {...args} />
+      <ReceivablesTable {...args} onTabChange={undefined} tab={undefined} />
+    </div>
+  ),
+};
+
+export const WithCustomTabs: Story = {
+  args: {
+    onRowClick: action('onRowClick'),
+  },
+  render: (args) => (
+    <div style={{ height: 500, padding: 20 }}>
+      <ExtendThemeProvider
+        theme={{
+          components: {
+            MoniteReceivablesTable: {
+              defaultProps: {
+                tabs: [
+                  {
+                    label: 'Draft Invoices',
+                    query: {
+                      type: 'invoice',
+                      sort: 'created_at',
+                      order: 'desc',
+                      status__in: ['draft'],
+                    },
+                    filters: [
+                      'document_id__contains',
+                      'counterpart_id',
+                      'due_date__lte',
+                    ],
+                  },
+                  {
+                    label: 'Recurring invoices',
+                    query: {
+                      type: 'invoice',
+                      status__in: ['recurring'],
+                    },
+                    filters: ['document_id__contains', 'counterpart_id'],
+                  },
+                  {
+                    label: 'Other Invoices',
+                    query: {
+                      type: 'invoice',
+                      sort: 'created_at',
+                      order: 'desc',
+                      status__in: [
+                        'accepted',
+                        'expired',
+                        'declined',
```

Comment:
```suggestion
                        'issued',
                        'overdue',
```
`accepted`, `declined`, and `expired` are quote statuses.